pkg/webservice/handlers_test.go

```diff
@@ -1068,6 +1068,12 @@ func TestGetQueueApplicationsHandler(t *testing.T) {
 	assert.NilError(t, err, "failed to unmarshal applications dao response from response body: %s", string(resp.outputBytes))
 	assert.Equal(t, len(appsDao), 2)
 
+	if !appsDao[0].HasReserved {
+		assert.Equal(t, len(appsDao[0].Reservations), 0)
+	} else {
+		assert.Check(t, len(appsDao[0].Reservations) > 0, "Get wrong reservation info from apps dao")
```

Comment:
Can we change the message to "reservation count should have been more than zero" or something like that?

Reply:
Thanks! It makes sense.